slack_bolt/app/app.py

```diff
@@ -664,6 +686,10 @@ def custom_error_handler(error, body, logger):
             logger=self._framework_logger,
             func=func,
         )
+        self._middleware_error_handler = CustomMiddlewareErrorHandler(
```

Comment:
For backward-compatibility, I didn't rename and modify the existing `CustomListenerErrorHandler` to be reusable for this. In the future major version, we may want to have a unified interface for these two error handlers. They are mostly the same.